frontend/src/auth/components/EmailVerificationPage/displayMessages.ts

```diff
@@ -0,0 +1,32 @@
+const messages = {
+  failedVerification: {
+    defaultMessage: "The verification code used is expired or doesn't exist.",
+    description: ''
+  },
+  thankYouMessage: {
+    defaultMessage: 'Thank you for verifying your email address.',
+    description: ''
+  },
+  followUp: {
+    defaultMessage:
+      'Your instance should be ready within the next 2 hours. ' +
+      'In the meantime, you can start customizing your instance.',
+    description: ''
+  },
+  contact: {
+    defaultMessage:
+      "At any point, don't hesitate to write to us at " +
+      'contact@opencraft.com if you have any questions or issues.',
```

Comment:
@giovannicimolin, can we not hard-code `contact@opencraft.com`?

Reply:
Done!